proto/ibc/lightclients/wasm/v1/query.proto

```diff
@@ -13,7 +13,7 @@ service Query {
     option (google.api.http).get = "/ibc/lightclients/wasm/v1/checksums";
   }
 
-  // Get Wasm code for given code hash
+  // Get Wasm code for given checksum
```

Comment:
Looks like there's an inconsistency on main here. I re-ran `make proto-all` when resolving conflicts which produced a change in the `.pb.go` godocs with checksum -> code hash. I believe it should be "checksum", right?